### PR TITLE
fix(webinstall): Correctly pass brew install command

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: tools/webinstall/install.sh [hunk 1]
```diff
@@ -1113,7 +1113,9 @@ install_darwin() {
 
     need_cmd "$BREW"
 
-    do_cmd "$BREW" install unikraft/cli/kraftkit
+    _ind_cmd=$(printf "%s%s%s" "$BREW" " install" " unikraft/cli/kraftkit")
+
+    do_cmd "$_ind_cmd"
```
Comment (nderjung) [inline tools/webinstall/install.sh]: Can this not be turned into a single string?
```bash
do_cmd "$BREW install unikraft/cli/kraftkit"
```
?

Comment (craciunoiuc) [reply tools/webinstall/install.sh]: I guess so, let me try

Comment (craciunoiuc) [reply tools/webinstall/install.sh]: yeah it works, whoops :D

**Review by nderjung — APPROVED** [review]
Thanks!

**Pull request merged**